doc/user_documentation.rst

```diff
@@ -144,6 +144,18 @@ It's even easier for a Fedora Rawhide package::
 because 'fedora' distgit is the default, and we automatically pick the default
 branch.
 
+.. note::
+    Please note that an SRPM is downloaded from the specified DistGit
+    only once per Copr build, regardless of the number of enabled
+    chroots. It **is not** the case that within one Copr build,
+    ``fedora-37-x86_64`` chroot would be built from the ``f37``
+    branch, ``fedora-38-x86_64`` from the ``f38`` branch, and
+    ``fedora-39-x86_64`` from the ``rawhide`` branch. You can,
+    however, use the **Committish** field to specify what DistGit
+    branch should be used (by default, it is ``rawhide`` for
+    `Fedora DistGit <https://src.fedoraproject.org/>`_).
```

Comment:
And typical work-around (to use different sources for different chroots) is to
submit multiple builds like:
```
$ copr build-distgit ping --name <package> --chroot fedora-rawhide-x86_64 --distgit fedora --commit rawhide
$ copr build-distgit ping --name <package> --chroot epel-8-x86_64         --distgit fedora --commit epel8
```

Reply:
Good idea, added.